examples/coherence/README.md

```diff
@@ -0,0 +1,21 @@
+# helidon-creditscore-se
```

Comment:
shouldn't it be named after 
```
<name>Helidon Examples Coherence SE</name>
```

?

Reply:
I reflected the changes in the MP example

Reply:
I reflected the changes in the MP example